python/ray/data/_internal/logging.py

```diff
@@ -2,15 +2,21 @@
 import logging.config
 import os
 from typing import Optional
-
 import yaml
 
 import ray
 
-DEFAULT_CONFIG_PATH = os.path.abspath(
+DEFAULT_TEXT_CONFIG_PATH = os.path.abspath(
     os.path.join(os.path.dirname(__file__), "logging.yaml")
 )
 
+DEFAULT_JSON_CONFIG_PATH = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), "logging_json.yaml")
+)
+
+# Environment variable name for to specify the encoding of the log messages
+RAY_DATA_LOG_ENCODING = os.environ.get("RAY_DATA_LOG_ENCODING", "TEXT")
```

Comment:
based on the logic below, i don't think we need to specify `"TEXT"` as the default encoding type, since text ends up being the default else case.
```suggestion
RAY_DATA_LOG_ENCODING = os.environ.get("RAY_DATA_LOG_ENCODING")
```